src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishASingleFileApp.cs

```diff
@@ -156,7 +156,7 @@ public void It_excludes_ni_pdbs_from_single_file()
             string[] expectedFiles = { SingleFile, PdbFile, NiPdbFile };
             GetPublishDirectory(publishCommand)
                 .Should()
-                .OnlyHaveFiles(expectedFiles);
+                .HaveFiles(expectedFiles);
```

Comment:
Do you think it might be worth adding this check also:
``` c#
foreach(file in output folder)
{
    if (file == SingleFile || file == PdbFile || file == NiPdbFile)
        continue;
    else
        Verify file.extension == ni.pdb
}
```

Reply:
LGTM otherwise

Reply:
I'm not sure it is worth tracking what `.ni.pdb` files are generated in this test -- it makes the test a bit fragile because of other files changing. 

Also,in the long term, I don't expect the mscorlib.ni.pdb (or any other ni.pdb) to be generated everytime. So, I'd keep the change as-is for now. Once the mscorlib compilation issue is resolved, we should move back to the `.OnlyHaveFiles` test.